src/clib/lib/enkf/model_config.cpp

```diff
@@ -238,22 +232,19 @@ model_config_type *model_config_alloc_empty() {
 
 model_config_type *model_config_alloc(const config_content_type *config_content,
                                       const char *data_root,
-                                      const ext_joblist_type *joblist,
                                       const ecl_sum_type *refcase) {
     model_config_type *model_config = model_config_alloc_empty();
 
     if (config_content)
-        model_config_init(model_config, config_content, data_root, 0, joblist,
-                          refcase);
+        model_config_init(model_config, config_content, data_root, 0, refcase);
```

Comment:
int ens_size is always zero, and isn't used.
Should we remove?

Reply:
Perhaps separate issue, but I can look into it.

Reply:
It is unused in the `model_config_init`.

Reply:
https://github.com/equinor/ert/pull/4205